### ISSUE TITLE
Add tutorials for new pipeline functions

### ISSUE THREAD

**Issue by bdpedigo** [microsoft/graspologic#820]
## Is your feature request related to a problem? Please describe.
Just wanted to document that we need tutorials/examples for the new pipeline functionality as we've discussed

## Related PRs
#814 #817 

## Additional context
cc @daxpryce

**Comment by rajpratyush** [comment]
should we create new sphinx based pages with examples ?

**Comment by daxpryce** [comment]
Yep, this is actually in process right now. @Nyecarr is building it out as we speak and I'm helping source and prep the data we're going to use.  We wanted to find one that would work well with both ASE and LSE, as well as be intuitive and human-reason-able so we can actually tell a compelling narrative for graph neophytes.

My current datasource we're expecting to use is Standford's https://snap.stanford.edu/data/wiki-topcats.html - but after I've translated the numeric IDs back to the string page titles, the file clocks in at ~ 900mb uncompressed.  I think doing this is important so that people can quickly read and reason about relationships without having to implicitly or explicitly map IDs to the string names.  But I also think that this is going to make documentation generation annoying and I'm thinking these will be pre-compiled notebooks... which may hide future errors.  It might just have to be a known risk /shrug
